README.md

```diff
@@ -82,10 +82,10 @@ $ minikube start --vm-driver=xhyve
 **Deploy FaaS-netes and the API Gateway**
 
 ```
-$ kubectl apply -f ./faas.yml,monitoring.yml
+$ kubectl apply -f ./faas.yml,monitoring-config.yml,monitoring.yml
```

Comment:
should we merge this into one?

Reply:
Prevents having to duplicate the config for arm and non arm

Reply:
OK makes sense. This will also need updating on the OpenFaaS K8s deployment guide.

Reply:
I see it, will do

Reply:
Hi Wei did you update the guide?

Reply:
(this is in the main FaaS repo) https://github.com/openfaas/faas/blob/master/guide/deployment_k8s.md

Reply:
yes it's in https://github.com/openfaas/faas/pull/287